packages/components/src/VirtualizedList/RowCollapsiblePanel/RowCollapsiblePanel.scss

```
@@ -11,4 +11,25 @@
 		margin-bottom: $padding-small;
 		margin-top: $padding-small;
 	}
+
+	.loading-collapsible-panel {
+		height: 40px;
```

```suggestion
		height: 4rem;
```

Please never use px if you can. 
With our them font-size : 1rem = 10px